packages/config/src/validation.ts

```diff
@@ -119,66 +119,99 @@ export const isValidBrowserList = (_key: string, browsers: any): ErrResult | tru
   return true
 }
 
-const isValidExperimentalRetryOptionsConfig = (options: any, strategy: 'detect-flake-but-always-fail' | 'detect-flake-and-pass-on-threshold'): boolean => {
-  if (options != null) {
-    // retries must be an integer of 1 or greater
-    const isValidMaxRetries = _.isInteger(options.maxRetries) && options.maxRetries > 0
+const isValidExperimentalRetryOptionsConfig = (key: string, value: any, strategy: 'detect-flake-but-always-fail' | 'detect-flake-and-pass-on-threshold') => {
+  if (value == null) return true
 
-    if (!isValidMaxRetries) {
-      return false
-    }
+  const isValidMaxRetries = isValidRetryValue(`${key}.maxRetries`, value.maxRetries, 1)
 
-    // if the strategy is 'detect-flake-but-always-fail', stopIfAnyPassed must be provided and must be a boolean
-    if (strategy === 'detect-flake-but-always-fail') {
-      if (options.passesRequired !== undefined) {
-        return false
-      }
+  if (isValidMaxRetries !== true) {
+    return isValidMaxRetries
+  }
 
-      const isValidStopIfAnyPasses = _.isBoolean(options.stopIfAnyPassed)
+  const validKeys = ['maxRetries']
+
+  // if the strategy is 'detect-flake-but-always-fail' and if stopIfAnyPassed is provided it must be a boolean
+  if (strategy === 'detect-flake-but-always-fail') {
+    validKeys.push('stopIfAnyPassed')
+    if (value.stopIfAnyPassed !== undefined) {
+      const isValidStopIfAnyPasses = _.isNull(value.stopIfAnyPassed) || _.isBoolean(value.stopIfAnyPassed)
 
       if (!isValidStopIfAnyPasses) {
-        return false
+        return errMsg(`${key}.stopIfAnyPassed`, value.stopIfAnyPassed, 'null or boolean')
```

Comment:
`stopIfAnyPassed` I believe can either be a `boolean` or `undefined` but `null` is not an option based on [1137](https://cypress-io.atlassian.net/browse/CYCLOUD-1137). If this has changed, we need to update the typings definition in the app in `cypress.d.ts` (which we might need to do anyway since the current implementation forces a user to specify it if they provide `experimentalOptions` on `detect-flake-but-always-fail`